demo/src/App.jsx

```diff
@@ -52,6 +53,7 @@ const links = {
   'parallax-vert': ParallaxVert,
   'parallax-sticky': ParallaxSticky,
   three: SimpleThree,
+  test: Test,
```

Comment:
Can we name this something more intuitive, I guess this is all about adding and subtracting springs? So something along that line.

Reply:
Do you think we should include this demo or just rely on the new tests? It's not really at the calibre of the rest of the demos, and I just grabbed it from a discord user who had posted this bug originally.

Reply:
From ericvu in #react-spring

Reply:
i think this demo app is super confusing since it's half-broken, do you think its okay to remove this demo app and just rely on the tests?